compiler/next/include/chpl/uast/Use.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2021 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CHPL_UAST_USE_H
+#define CHPL_UAST_USE_H
+
+#include "chpl/queries/Location.h"
+#include "chpl/uast/Decl.h"
+#include "chpl/uast/Expression.h"
+#include "chpl/uast/UseClause.h"
+
+namespace chpl {
+namespace uast {
+
+
+/**
+  This class represents a use statement. For example:
+
+  \rst
+  .. code-block:: chapel
+
+      // Example 1:
+      use Foo, Bar as A;
+
+  \endrst
+
+  This creates a use statement that has two use clauses, 'Foo' and
+  'Bar as A'.
+*/
+class Use final : public Expression {
+ private:
+  Use(ASTList children, Decl::Visibility visibility)
+    : Expression(asttags::Use, std::move(children)),
+      visibility_(visibility) {
+    assert(numChildren() >= 1);
+    assert(isExpressionASTList(children_));
```

Comment:
Child expressions can only be `UseClause`, right? Can you add a function to call in an assert to check that? I have added such a pattern several times, see e.g. https://github.com/chapel-lang/chapel/blob/master/compiler/next/include/chpl/uast/MultiDecl.h#L59